recipe/conda_build_config.yaml

```diff
@@ -557,7 +557,7 @@ sox:
 sqlite:
   - 3
 sundials:
-  - 3.2 
+  - 3.2.*
```

Comment:
If I understand this correctly the `3.2` pin and `3.2.*` will be the same thing b/c the pin is the minor version (`x.x`), right? If so, can we close this one?

Reply:
yes, I guess so